README.md

```diff
@@ -19,33 +19,35 @@ From these Go sources, several files are generated:
 
 Generated files are created by a build script (see section [How to build](#how-to-build)).
 
-### Devfile 2.0.0 file format
+## Devfile 2.0.0 file format
 
 A Subset of this `DevWorkspace` API defines a structure (workspace template content), which is also at the core of the **Devfile 2.0** format specification.
-For more information about this, please look into the [Devfile support Readme](devfile-support/README.md)
+For more information about this, please look into the [Devfile support README]([devfile-support/README.md](https://github.com/devfile/registry-support/blob/main/README.md))
 
-The generated documentation of the Devfile 2.0 format, based on its json schema, is available here: https://devfile.github.io
+The generated documentation of the Devfile 2.0 format, based on its json schema, is available here: <https://devfile.github.io>
 
-Typescript model is build on each commit of main branch and available as NPM package at https://www.npmjs.com/package/@devfile/api
+Typescript model is build on each commit of main branch and available as NPM package at <https://www.npmjs.com/package/@devfile/api>
 
 ## Release
+
 Release details and process are found in [Devfile Release](RELEASE.md)
 
 ## How to build
 
 In order to build the CRD and the various schemas, you don't need to install any pre-requisite apart from `docker`.
 In the root directory, just run the following command:
 
-```
-./docker-run.sh ./build.sh
+```console
+bash ./docker-run.sh ./build.sh
```

Comment:
Is `bash` necessary here? The scripts have the bash hashbang at the top (`#!/bin/bash`)

Reply:
On non linux systems you can run into issues where you have permission to execute `bash ./*.sh` but not `./*.sh`

Reply:
Makes sense